pipenv/cli.py

```diff
@@ -407,6 +417,8 @@ def do_create_virtualenv(three=None, python=None):
             python = 'python2'
         elif three is True:
             python = 'python3'
+        else:
+            python = get_python_from_requires()
```

Comment:
This is only usable with `pipenv install` which encourages a pattern that's been problematic in the past. It's usually a much better idea to initialize your environment before running the install command.

The unpredictability of the outcome here is also a negative in my eyes. If the user has to run `pipenv run python --version` to confirm that this didn't silently fail over to the system default python, then we're not saving much time. It may work well in highly controlled images but I don't think that's the average users setup. I'd lean back on `explicit is better than implicit` here and say it's just as easy, and more predictable, to pass `--python pythonX.X` with your install command.